dev/ci/yarn-install-with-retry.sh

```diff
@@ -10,7 +10,7 @@ while : ; do
     echo "--- ✂️ timeout reached, aborting".
     exit 1
   fi
-  if yarn --immutable --network-timeout 30000 "$@" 2> >(tee "$tmp_log">&2); then
+  if yarn install --network-timeout 30000 "$@" 2> >(tee "$tmp_log">&2); then
     break
   fi
 
```

Comment:
Should we delete this file?

Reply:
I think it was still used in one place that I haven't updated (I think I wasn't sure how). Looks like I did start converting this one and didn't fully revert it though.